### ISSUE TITLE
JetStream `max_file_store` config setting misnamed `max_file_storage` in the documentation

### ISSUE THREAD

**Issue by davidkemper** [ConnectEverything/synadia-website#83]
See https://docs.nats.io/running-a-nats-service/configuration#jetstream

**Comment by gcolliso** [comment]
@davidkemper so the setting just needs to be changed to `max_file_store` correct?

**Comment by wallyqs** [comment]
yes,  I think better to change to `max_file_store` so that the example config below

It can be either `"max_file_store"` or  `"max_file"`:
https://github.com/nats-io/nats-server/blob/9d6525c8a3a5470da3d98570936caca79319e17a/server/opts.go#L1806
